embassy-stm32/src/usb/otg.rs

```diff
@@ -103,15 +103,18 @@ impl<'d, T: Instance> Driver<'d, T> {
     pub fn new_hs(
         _peri: impl Peripheral<P = T> + 'd,
         _irq: impl interrupt::typelevel::Binding<T::Interrupt, InterruptHandler<T>> + 'd,
-        dp: impl Peripheral<P = impl DpPin<T>> + 'd,
-        dm: impl Peripheral<P = impl DmPin<T>> + 'd,
+        _dp: impl Peripheral<P = impl DpPin<T>> + 'd,
```

Comment:
Is there a reason to apply the `_` naming here? Usually that is done for objects that are unused but you call `set_as_ref` on them or am I missing something?

Reply:
Dunno. Rust language limit hit. For the case the pins are analogue these values are not referenced, so added the _. If there's a better way, please advise.

Reply:
Oh I totally missed the feature flag there to keep those pins in analog mode which would result in them not be used and rust complaining. My bad.